### PR TITLE
User Login+Session

### PR DESCRIPTION
**Pull request by jonathandkim96**
TODO: rest of users API

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,5 +1,10 @@
+//delete later
+import usersFunctions from "./data/users.js";
+//
```
Comment (sanjeet-jain) [inline app.js:3]: remove unwanted code

File: app.js [hunk 2]
```diff
@@ -22,11 +27,58 @@ app.use("/public", express.static(__dirname + "/public"));
...
+    {first_name:"reese",
+    last_name:"puffs",
+    email:"hahaha@lol.com",
+    Disability:false,
+    Dob:"08/02/2001"}
+  )
+  console.log(deez);
+}catch(e){
+  console.log(e);
+}*/
```
Comment (sanjeet-jain) [inline app.js:69]: remove this too

File: app.js [hunk 3]
```diff
@@ -22,11 +27,58 @@ app.use("/public", express.static(__dirname + "/public"));
...
+/*
+app.use("/login", (req, res, next) => {
+  console.log(req.session.id);
+  if (req.session.user) {
+    return res.redirect('/');
+  } else {
+    next();
+  }
+});
+*/
```
Comment (sanjeet-jain) [inline app.js:81]: remove

File: data/users.js [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+const exportedMethods = {
+    async create(
+        first_name,
+        last_name,
+        email,
+        password,
+        Disability,
+        Dob,
+        Consent
```
Comment (sanjeet-jain) [inline data/users.js:32]: make lower case

File: routes/users.js [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+import usersFunctions from "../data/users.js";
+import utils from "../utils/utils.js";
+
+
+
+router
+    .route("/signup")
+    .post(async(req,res) => {
+        //validate
```
Comment (sanjeet-jain) [inline routes/users.js:13]: add as a TODO comment //TODO

File: tasks/seed.js [hunk 6]
```diff
@@ -21,13 +22,13 @@ export async function runSetup() {
...
   const sampleUser = {
     first_name: "Sample",
     last_name: "User",
     email: "sampleUser@gmail.com",
     // password will be hashed when being passed from UI to the API and then stored ( done by Jonathan)
-    password: "sampleUser",
+    password: hashPW,
     disability: false,
     // date string passed here is MM/DD/YYYY
     dob: new Date("01/01/1996"),
```
Comment (sanjeet-jain) [inline tasks/seed.js:34]: doesnt this need to be a string ? 
lets discuss

**Review by sanjeet-jain — CHANGES_REQUESTED** [review]
please fix the issues

File: app.js (hunk 1 above)
Comment (jonathandkim96) [reply app.js]: done

File: app.js (hunk 2 above)
Comment (jonathandkim96) [reply app.js]: done

File: data/users.js (hunk 4 above)
Comment (jonathandkim96) [reply data/users.js:32]: i will do it in the next iteration

File: tasks/seed.js (hunk 6 above)
Comment (jonathandkim96) [reply tasks/seed.js:34]: i like bread

File: app.js (hunk 3 above)
Comment (jonathandkim96) [reply app.js]: done

File: routes/users.js (hunk 5 above)
Comment (jonathandkim96) [reply routes/users.js:13]: to do

**Review by sanjeet-jain — APPROVED** [review]

**Pull request merged**